.github/workflows/windows.yml

```diff
@@ -47,24 +62,8 @@ jobs:
           docs/*.md
     - name: test
       run: |
-        if ("${{matrix.configuration}}" -eq "Debug") {
-          $testCategory = "smoke";
-        }
-        elseif("${{matrix.platform}}" -eq "x64") {
-          $testCategory = "full";
-        }
-        else {
-          $testCategory = "quick";
-        }
-        if ("${{matrix.platform}}" -eq "Win32") {
-          $testPlatform = "x86";
-        }
-        else {
-          $testPlatform = "x64";
-        }
-        $slangTestBinDir = ".\bin\windows-$testPlatform\${{matrix.configuration}}\";
+        $slangTestBinDir = ".\bin\windows-${{matrix.testPlatform}}\${{matrix.configuration}}\";
         $env:Path += ";$slangTestBinDir";
-        Invoke-WebRequest -uri "https://github.com/shader-slang/swiftshader/releases/download/v1.0/vk_swiftshader_windows_$testPlatform.zip" -Method "GET"  -Outfile "swiftshader.zip";
-        Expand-Archive "swiftshader.zip" -DestinationPath $slangTestBinDir;
-        & "$slangTestBinDir\slang-test.exe" -api all-dx12 -appveyor -bindir "$slangTestBinDir\" -platform $testPlatform -configuration ${{matrix.configuration}} -category $testCategory 2>&1;
+        Expand-Archive "vk_swiftshader_windows_${{matrix.testPlatform}}.zip" -DestinationPath $slangTestBinDir;
+        & "$slangTestBinDir\slang-test.exe" -api all-dx12 -appveyor -bindir "$slangTestBinDir\" -platform $testPlatform -configuration ${{matrix.configuration}} -category ${{matrix.testCategory}} 2>&1;
```

Comment:
There is a `$testPlatform` here that needs to be replaced with `${{matrix.testPlatform}}`.